test/integration/Middleware.t.sol

```diff
@@ -969,7 +979,7 @@ contract MiddlewareTest is Test {
                     tanssi.subnetwork(0), _operator, depositAmount
                 );
             }
-            middleware.registerOperator(_operator, bytes32(uint256(i + 4)));
+            middleware.registerOperator(_operator, abi.encode(bytes32(uint256(i + 4))), address(0));
```

Comment:
Could we put this into a variable with a descriptive name. Pretty hard to know what this is about at first glance:
`abi.encode(bytes32(uint256(i + 4)))`